src/themes/proficiency.css

```diff
@@ -201,6 +201,13 @@ body[data-theme="proficiency"] {
       width: 100%;
     }
   }
+
+  .question-container, .dashboard-container {
+    /* WORKAROUND: hide the table background, while keeping popover backgrounds white */
+    [role="gridcell"], [role="columnheader"], [data-testid="TableInteractive-root"], [aria-label="grid"] {
+      background: transparent;
+    }
+  }
```

Comment:
Hides the table background color, while keeping popovers and tooltips intact.

![CleanShot 2568-01-14 at 23 03 56@2x](https://github.com/user-attachments/assets/47118a36-3ad5-4c21-b9b6-3d62cd32d365)